projects/frontend/data-pipelines/gui/package.json

```diff
@@ -7,16 +7,17 @@
   },
   "private": true,
   "scripts": {
-    "ng": "node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng",
-    "start": "npm run ng -- serve",
-    "build": "npm run ng -- build --configuration=production",
+    "ng:sc": "node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng",
```

Comment:
what is sc?

Reply:
"ng:sc" is just simple shorthand script in context of package.json that could be used as a base script to run "ng" command with node flags and append other commands like bellow
```
npm run ng:sc -- build --configuration=production
```

Reply:
Why is this change needed??

Reply:
I just reverted to script with name "ng" and I'm totally fine with not introducing it